src/main.rs

```diff
@@ -339,6 +353,7 @@ fn merge_avgs(
             severity: avg_severity,
             status: avg_status,
             required_by: vec![],
+            avg_type: avg_types.iter().join(","),
```

Comment:
I would rather store them as a list here and use `join()` when printing them.

Reply:
done